README.md

```diff
@@ -70,19 +70,14 @@ Both the packages below provide a convenience macro to provide einstein summatio
 * [Einsum.jl](https://github.com/ahwillia/Einsum.jl)
 * [TensorOperations.jl](https://github.com/Jutho/TensorOperations.jl)
 
-[docs-latest-img]: https://img.shields.io/badge/docs-latest-blue.svg
-[docs-latest-url]: https://kristofferc.github.io/Tensors.jl/latest/
 
 [docs-stable-img]: https://img.shields.io/badge/docs-stable-blue.svg
 [docs-stable-url]: https://kristofferc.github.io/Tensors.jl/stable
 
-[travis-img]: https://travis-ci.org/KristofferC/Tensors.jl.svg?branch=master
-[travis-url]: https://travis-ci.org/KristofferC/Tensors.jl
+[ci-img]: https://github.com/Ferrite-FEM/Tensors.jl/workflows/CI/badge.svg
```

Comment:
```suggestion
[ci-img]: https://github.com/Ferrite-FEM/Tensors.jl/workflows/CI/badge.svg?branch=master
```

Reply:
Is master not the default? What is it showing otherwise?

Reply:
Somebody told me that it might interfere with the most recent commit to some PR, but I don't know.